tests-utils/src/main/scala/tech.beshu.ror.utils/containers/ReadonlyRestEsClusterContainer.scala

```diff
@@ -185,15 +184,13 @@ class ReadonlyRestEsRemoteClustersContainer private[containers](val localCluster
   }
 
 }
-
-trait ReadonlyRestEsClusterInitializer {
-  def initialize(adminClient: RestClient, container: ReadonlyRestEsClusterContainer): Unit
-}
-
-object NoOpReadonlyRestEsClusterInitializer extends ReadonlyRestEsClusterInitializer {
-  override def initialize(adminClient: RestClient, container: ReadonlyRestEsClusterContainer): Unit = ()
+final case class RorNodeSpecification(envs:Map[String,String])
+object RorNodeSpecification {
+  implicit val monoidRorNodeSpecification: Monoid[RorNodeSpecification] = {
```

Comment:
sorry, it doesn't make sense for me. `Monoid` should be used when there is a structure which can be composed. I don't see where we'd like to use composition of `RorNodeSpecification`. So, we end up with usage monoid `empty`, additional dependency and far-fetched usage of Monoid concept. Please, get rid of it.

Reply:
When using `Map` it's not monoid, but with `List` it will become monoid. It is composable structure. For now it supports only environment variables, but it can support more. Alternative was adding node envs list to AdditionalClusterSettings, not encapsulating this specific, low level setting.

Reply:
But I don't see usage of RorNodeSpecification as monoid. Now and in future. Please, remove this unnecessary complication.